sandbox/mutual_rec.eo

```diff
@@ -0,0 +1,19 @@
++package sandbox
```

Comment:
@Sitiritis the file is here, but it's not being used anyhow, not even mentioned in the `README`. Maybe we should mention it?